src/Common/src/Interop/Interop.UnsafeNativeMethods.Odbc.cs

```diff
@@ -1,15 +1,16 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The file should just be Interop.Odbc.cs, named to match whatever you name the class (Odbc or Odb32).